docker-compose.yml

```
@@ -0,0 +1,402 @@
+    networks:
+      - kafka-network
+
+  redis:
+    image: redis:7-alpine
+    container_name: antcamp-redis
+    ports:
+      - "${SPRING_DATA_REDIS_PORT}:6379"
+    volumes:
+      - redisdata:/data
+    healthcheck:
+      test: [ "CMD", "redis-cli", "ping" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)